projects/1.20/assets/storage-drawers/storagedrawers/lang/zh_cn.json

```
@@ -0,0 +1,142 @@
+{
+    "block.storagedrawers.type.full_drawers_1": "%s 1x1基础抽屉",
+    "block.storagedrawers.type.full_drawers_2": "%s 1x2基础抽屉",
+    "block.storagedrawers.type.full_drawers_4": "%s 2x2基础抽屉",
+    "block.storagedrawers.type.half_drawers_1": "%s 1x1小型基础抽屉",
+    "block.storagedrawers.type.half_drawers_2": "%s 1x2小型基础抽屉",
+    "block.storagedrawers.type.half_drawers_4": "%s 2x2小型基础抽屉",
+    "block.storagedrawers.type.trim": "%s 桥接方块",
+  
+    "block.storagedrawers.mat.oak": "橡木",
+    "block.storagedrawers.mat.spruce": "云杉木",
+    "block.storagedrawers.mat.birch": "白桦木",
+    "block.storagedrawers.mat.jungle": "丛林木",
+    "block.storagedrawers.mat.acacia": "金合欢木",
+    "block.storagedrawers.mat.dark_oak": "深色橡木",
+    "block.storagedrawers.mat.mangrove": "红树木",
+    "block.storagedrawers.mat.cherry": "樱花木",
+    "block.storagedrawers.mat.bamboo": "竹子",
+    "block.storagedrawers.mat.crimson": "绯红木",
+    "block.storagedrawers.mat.warped": "诡异木",
+    "block.storagedrawers.mat.framed": "镶框",
+  
+    "block.storagedrawers.compacting_drawers_2": "压缩抽屉（2级）",
+    "block.storagedrawers.compacting_drawers_3": "压缩抽屉（3级）",
+    "block.storagedrawers.compacting_half_drawers_2": "小型压缩抽屉（2级）",
+    "block.storagedrawers.compacting_half_drawers_3": "小型压缩抽屉（3级）",
+    "block.storagedrawers.framed_compacting_drawers_2": "镶框压缩抽屉（2级）",
+    "block.storagedrawers.framed_compacting_drawers_3": "镶框压缩抽屉（3级）",
+    "block.storagedrawers.framed_compacting_half_drawers_2": "镶框小型压缩抽屉（2级）",
+    "block.storagedrawers.framed_compacting_half_drawers_3": "镶框小型压缩抽屉（3级）",
+    "block.storagedrawers.controller": "抽屉管理器",
```

```suggestion
    "block.storagedrawers.controller": "抽屉控制器",
```
后方统一，也可选择修改1.16、1.18、1.19版本